python-squid/src/test/java/org/sonar/python/FileLinesVisitorTest.java

```diff
@@ -45,30 +43,25 @@
 
   private static final File BASE_DIR = new File("src/test/resources/metrics");
 
-  @Mock
-  private Project project;
-  @Mock
-  private FileLinesContextFactory fileLinesContextFactory;
-  @Mock
-  private ProjectFileSystem fileSystem;
-
   @Before
   public void before() {
     MockitoAnnotations.initMocks(this);
```

Comment:
MockitoAnnotations.initMocks() is not needed any more if no mock annotation is used.